generated/nidigital/nidigital/_library.py

```diff
@@ -8,6 +8,8 @@
 
 import nidigital.history_ram_cycle_information as history_ram_cycle_information  # noqa: F401
 
+import nidigital.capture_waveform as capture_waveform  # noqa: F401
```

Comment:
For my own curiosity, why do we do import this is we don't use it?

Reply:
Unlike the custom type in this PR, some types like:
https://github.com/ni/nimi-python/blob/397d60511f77852dba4494420dee8e343492fc5d/generated/niscope/niscope/waveform_info.py#L9
are passed in/out of the C library. We don't have a way to differentiate between the two kinds in metadata, so we unconditionally add them to `_library`. We could add something to metadata but we try to keep metadata keys as minimal as possible.